src/installer/corehost/cli/hostmisc/pal.unix.cpp

```diff
@@ -74,11 +74,16 @@ void* pal::map_file_readonly(const pal::string_t& path, size_t& length)
         close(fd);
         return nullptr;
     }
+    size_t size = buf.st_size;
```

Comment:
`size_t` is only for size in memory; for files, you want `off_t`.

Reply:
I agree that `buf.st_size` is an `off_t` but we're computing `size` to be an input for `mmap()`, which expects a `size_t length` argument .